### DISCUSSION TITLE
Center button rows on "entities" cards

### DISCUSSION THREAD

**Discussion by jlpouffier** [home-assistant/frontend#10892] [Dashboard] [answered]
Hello,

I quite like the new "buttons" when you create button rows on the "entities" card.

Feature idea: Could we have a way to center them?

Before they were centered by default, as such
![image](https://user-images.githubusercontent.com/5878296/145798819-bc2695ae-4af0-4691-a17f-14d5070a665f.png)

But now they are left-aligned by default, with no way of changing this behavior. As such 
![image](https://user-images.githubusercontent.com/5878296/145798919-27ca996b-e1bd-45c8-862c-cf56a5a0b44a.png)

I would like to have a way to center them again.

Thank you

**Comment by spacegaier** [comment, marked as answer]
Sounds like https://github.com/home-assistant/frontend/issues/10800

**Comment by mneuron** [comment]
See my quick fix: https://github.com/home-assistant/frontend/issues/10800#issuecomment-992459897

![image](https://user-images.githubusercontent.com/17382871/145817100-5e21e125-f844-4d84-bea3-7347ec210fe9.png)
